### ISSUE TITLE
Running 'make dev' causes changes to vendored packages

### ISSUE THREAD

**Issue by DanHam** [hashicorp/packer#6152]
Packer version: Latest
Host platform: OS X 10.11.6 (15G20015)

When I run `make dev` on the master branch I get a number of changes due to the running of `govendor sync`

```bash
[master] $ git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean

[master] $ make dev

[master] $ git status
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	deleted:    vendor/github.com/sirupsen/logrus/terminal_check_appengine.go
	modified:   vendor/vendor.json

Untracked files:
  (use "git add <file>..." to include in what will be committed)

	vendor/github.com/sirupsen/logrus/CHANGELOG.md
	vendor/github.com/sirupsen/logrus/LICENSE
	vendor/github.com/sirupsen/logrus/README.md
	vendor/github.com/sirupsen/logrus/alt_exit.go
	vendor/github.com/sirupsen/logrus/doc.go
	vendor/github.com/sirupsen/logrus/entry.go
	vendor/github.com/sirupsen/logrus/exported.go
	vendor/github.com/sirupsen/logrus/formatter.go
	vendor/github.com/sirupsen/logrus/hooks.go
	vendor/github.com/sirupsen/logrus/json_formatter.go
	vendor/github.com/sirupsen/logrus/logger.go
	vendor/github.com/sirupsen/logrus/logrus.go
	vendor/github.com/sirupsen/logrus/terminal_bsd.go
	vendor/github.com/sirupsen/logrus/terminal_linux.go
	vendor/github.com/sirupsen/logrus/text_formatter.go
	vendor/github.com/sirupsen/logrus/writer.go

no changes added to commit (use "git add" and/or "git commit -a")
 ```

Just running `govendor sync` in the root packer directory causes the same thing...

I also noticed we have:
`vendor/github.com/Sirupsen/logrus` and
`vendor/github.com/sirupsen/logrus`

**Comment by mwhooker** [comment]
looks like the case was changed in f6135a440c283b88caaf3314f10272037cfe1a4f, which is regrettable, but shouldn't usually be a problem. What does `git config --get core.ignorecase` say?

**Comment by DanHam** [comment]
`git config --get core.ignorecase` gives no output...

**Comment by mwhooker** [comment]
how did you check out packer?

**Comment by mwhooker** [comment]
fwiw:

> core.ignoreCase
If true, this option enables various workarounds to enable Git to work better on filesystems that are not case sensitive, like FAT. For example, if a directory listing finds "makefile" when Git expects "Makefile", Git will assume it is really the same file, and continue to remember it as "Makefile".

> **The default is false, except git-clone[1] or git-init[1] will probe and set core.ignoreCase true if appropriate when the repository is created.**

emphasis mine

**Comment by DanHam** [comment]
In all honesty I can't remember... it was way back.

**Comment by mwhooker** [comment]
osx uses a case insensitive _but_ space preserving fs. I know, a nightmare. I think it's probably a good idea to set `git config --local core.ignorecase false` inside the packer directory, but I also agree this is an issue. will fix cases in the vendor and push up

**Comment by mwhooker** [comment]
12f80208b

**Comment by mwhooker** [comment]
and 678f0647ccb333f35914979b4c4bbbb056893306

**Comment by DanHam** [comment]
I formatted with `Case-sensitive Journaled HFS+`... Case insensitivity on a *nix system just seemed weird to me...

Thanks for the quick fix!

**Comment by mwhooker** [comment]
well that makes sense then.

however I seem to have broken the build

**Comment by DanHam** [comment]
yup! 😄 I think the lowercase option is the right one, although previously we have uppercase...

**Comment by mwhooker** [comment]
fml https://github.com/sirupsen/logrus/issues/543

**Comment by mwhooker** [comment]
fixed 71abcdaab

**Comment by DanHam** [comment]
Ah... That case change caused a bit of upset then...

Well, fixed now as you say. Cheers!
